lua/astrocommunity/editing-support/nvim-paredit/README.md

```diff
@@ -0,0 +1,9 @@
+## nvim-paredit
+
+A Paredit implementation for Neovim, built using Treesitter and written in Lua.
+
+Supported Languages: Clojure, Fennel, Scheme, CommonLisp
+
+**Repository:** <https://github.com/julienvincent/nvim-paredit>
+
+> NOTE: nvim-paredit is included in the Clojure pack
```

Comment:
I cant seem to find paredit in the clojure pack.

Reply:
The intent is to replace nvim-treesitter-sexp with nvim-pareidt, but use parpar-nvim to provide nvim-paredit so it works well with nvim-parinfer.

I'll remove this line.

Reply:
Oh, its already changed.

Reply:
So shouldnt this be removed?

Reply:
Lets leave it off